.github/workflows/_build_test.yml

```diff
@@ -29,21 +32,14 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
-
-      - run: ./build.sh ${{ matrix.component }}
-
-      # TODO re-enable as soon as we make use of it
-      # - name: Scan Image
-      #   uses: ./.github/actions/scan-image
-      #   continue-on-error: true
-      #   with:
-      #     image_name: ${{ matrix.component }}
-
-      - uses: ./.github/actions/push-ghcr
+  
+      - uses: ./.github/actions/setup-environment
         with:
-          docker_image: wikibase/${{ matrix.component }}
           github_token: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Build and push ${{ matrix.imageName }}
+        run: pnpm exec nx build ${{ matrix.imageName }} --push
```

Comment:
Where is this `--push` coming from? I cannot find who defines it where?

Reply:
Any extra args to the `nx build` command / `build/build.sh` get forwarded to `docker buildx build . <here>`: https://docs.docker.com/reference/cli/docker/buildx/build/#push

Reply:
Got it, thank you